test/integration/features/cmd-openshift.feature

```diff
@@ -0,0 +1,114 @@
+      When executing "minishift openshift version" succeeds
+      Then stdout should match
+      """
+      ^openshift v[0-9]+\.[0-9]+\.[0-9]+\+[0-9a-z]{7}
+      kubernetes v[0-9]+\.[0-9]+\.[0-9]+\+[0-9a-z]{10}
+      etcd [0-9]+\.[0-9]+\.[0-9]+
+      """
+
+  Scenario: Getting address of internal docker registry
+  Prints the host name and port number of the OpenShift registry to the standard output.
+     Given Minishift has state "Running"
+      When executing "minishift openshift registry" succeeds
+      Then stdout should be valid IP with port number
+
+  Scenario: Getting existing service without route
+      When executing "minishift openshift service nodejs-ex" succeeds
+      Then stdout should contain "nodejs-ex"
+       And stdout should not match
+       """
+       ^http:\/\/nodejs-ex-myproject\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.nip\.io
+       """
+
+  Scenario: Getting non-existing service
+  If service does not exist, user gets an empty table.
+      When executing "minishift openshift service not-present" succeeds
+      Then stdout should not contain "not-present"
+
+  Scenario: Getting service from non-existing namespace
+      When executing "minishift openshift service nodejs-ex --namespace does-not-exist" fails
+      Then stderr should contain "Namespace does-not-exist doesn't exist"
+
+  Scenario: Forgotten service name
+      When executing "minishift openshift service --namespace myapp" fails
+      Then stderr should contain "You must specify the name of the service."
+
+  Scenario: User creates route to the service
+      When executing "oc expose svc/nodejs-ex" succeeds
+      Then stdout should contain
+      """
+      route "nodejs-ex" exposed
+      """
+
+  Scenario: Getting existing service with route
+      When executing "minishift openshift service nodejs-ex" succeeds
+      Then stdout should contain "nodejs-ex"
+       And stdout should match
+       """
+       http:\/\/nodejs-ex-myproject\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.nip\.io
+       """
+
+  Scenario: Getting URL of service using --url flag
+      When executing "minishift openshift service nodejs-ex --url" succeeds
+      Then stdout should be valid URL
+
+  Scenario: Seeing configuration of OpenShift master
+  Minishift openshift config view prints YAML configuration of OpenShift cluster.
+  Note: --target=master is default value for minishift openshift config command
+      When executing "minishift openshift config view" succeeds
+      Then stdout should be valid YAML
+
+#  Scenario: Seeing configuration of OpenShift node
+#      When executing "minishift openshift config view --target node" succeeds
+#      Then stdout should be valid YAML#
+
+  Scenario: Setting configuration on OpenShift master
+      When executing »minishift openshift config set --patch '{"assetConfig": {"logoutURL": "http://www.minishift.io"}}'« succeeds
+      Then stdout should contain "Patching OpenShift configuration"
```

Comment:
+1 Actually I was thinking about making code around JSON config file to be YAML compatible also. And to refactor it a little bit so it could also extract values from stdout, stderr etc.